pkg/services/ngalert/notifier/alertmanager_config.go

```diff
@@ -134,5 +134,16 @@ func (moa *MultiOrgAlertmanager) mergeProvenance(ctx context.Context, config def
 		}
 		config.Route.Provenance = provenance
 	}
+	cpProvs, err := moa.ProvStore.GetProvenances(ctx, org, "contactPoint")
```

Comment:
Rather than hardcoding the type ID, could we grab the type ID off the object?
```suggestion
	cpProvs, err := moa.ProvStore.GetProvenances(ctx, org, provisioning.EmbeddedContactPoint{}.ResourceType())
```

That way, our trick of storing strings identifying the type of object being provisioned is abstracted. And, we only have it defined in one place.

If we really wanted to be fancy, we could update `GetProvenances` to take a `Provisionable` instead of a string. Then, it's type safe because you can't pass in any random string or object. The call would look like `GetProvenances(ctx, org, provisioning.EmbeddedContactPoint{})` and it would return all provenances for embedded contact points. Maybe not necessary to go THAT far, though.